docker/build/internal_files/entrypoint.sh

```diff
@@ -48,9 +48,16 @@ EOF
 
     export NGINX_APISERVER_ADDR=${NGINX_APISERVER_ADDRESS:-http://apiserver:8008}
     export NGINX_FILESERVER_ADDR=${NGINX_FILESERVER_ADDRESS:-http://fileserver:8081}
-
     envsubst '${NGINX_APISERVER_ADDR} ${NGINX_FILESERVER_ADDR}' < /etc/nginx/clearml.conf.template > /etc/nginx/nginx.conf
 
+    if [[ -n "${CLEARML_SERVER_SUB_PATH}" ]]; then
+      envsubst '${CLEARML_SERVER_SUB_PATH}' < /etc/nginx/clearml_subpath.conf.template > /etc/nginx/default.d/clearml_subpath.conf
+      cp /usr/share/nginx/html/env.js /usr/share/nginx/html/env.js.origin
```

Comment:
Also, we should probably have `CLEARML_SERVER_SUB_PATH` documented somehow (even as a commented-out option in the docker compose file?)

Reply:
The problem is that all three components (web, api and files) are deployed in the same image. I think that you have first to validate this [PR](https://github.com/allegroai/clearml-web/pull/21).

Reply:
The purpose of `CLEARML_SERVER_SUB_PATH` variable is to give the possibility of deploying the application on a path other than "/". Whoever sets this variable knows that otherwise there is no reason to set it.
I completely agree with you, this information must be documented.

Reply:
Done.
I will add also in the helm chart doc.